jsx/Form.js

```diff
@@ -168,6 +167,7 @@ class SearchableDropdown extends React.Component {
     if (!this.props.strictSearch && value === undefined) {
       value = e.target.value;
     }
+    this.props.onUserInput(this.props.name + 'Temp', e.target.value);
```

Comment:
I don't understand what this "Temp" thing is all about in this PR.

Reply:
there needs to be an intermediary "temporary" value for the user's input to be stored in as they're typing. couldn't think of a better naming convention. open to any suggestions